src/components/Search/Filters/FilterBar.tsx

```diff
@@ -0,0 +1,17 @@
+import { filterTypeOptions, TypeKeys } from './FilterOptions'
+import FilterButton from './FilterButton'
+import styles from './FilterBar.module.css'
+
+export default function FilterBar() {
+  const options = Object.entries(filterTypeOptions)
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.filters}>
+        {options.map(([type, option], i) => (
+          <FilterButton key={i} type={type as TypeKeys} option={option} />
```

Comment:
issue/suggestion(blocking): Having to cast type to `TypeKeys` after importing it seems like unecessary. We probably should provide a dedicated interface here to simplify integration in other components.
Quick suggestion would be something reusable (for example in `FilterOptions.tsx`:
```jsx
export interface IFilterTypeOptions {
  [key: string]: IFilterTypeOpt
}

export interface IFilterOption {
  display?: string
  options: {
    value: string
    display?: string
    default?: boolean
  }[]
}
```
This also showcases another issue I see in the current implementation: the `display` prop currently seems unecessary or at leasrt optional. Please refer to the dedicated comment.